guides/v2.3/cloud/project/project-upgrade.md

```diff
@@ -27,6 +27,8 @@ If you use PHP version 7.2, you must remove the `mcrypt` extension from the [`ex
     ```bash
     composer require magento/magento-cloud-metapackage <requiredversion> --no-update
     ```
+    
+    > Be sure to use the `">=2.3.1 <2.3.2"` syntax for the value of `<requiredversion>`. This is necessary, as failing to do so will not properly update the ECE Tool.
```

Comment:
Please, rename "ECE Tool" to "ECE Tools".

Reply:
I can understand why you would want this note here.  I recommend removing that note. I updated the step instead. I also suggested providing the actual command, so it is easy for others to copy.

Reply:
i think the note is needed -- it's important to know why you would need to do this rather than blindly copying/pasting something. if this note wasn't there i would most likely just use the single version method.